### PR TITLE
feat: update home page bullet links UA-238

### PR DESCRIPTION
**Pull request by ghost**
- [x] https://geospoc.atlassian.net/browse/UA-238

### CODE REVIEW

File: content/en/1.index.md [hunk 1]
```diff
@@ -11,14 +11,14 @@ bulletLinks:
...
+        crowdSourceValidationPlatform: 'https://gqi-mapper-csp-prod.geospoc.com/',
       },
     'hospitals':
       {
-        codebase: '#',
-        documentation: '#',
-        portal: 'https://gqi-mapper-frontend-test.geospoc.io/dashboard',
-        crowdSourceValidationPlatform: 'https://gqi-mapper-csp-test.geospoc.io/',
+        codebase: 'https://github.com/geospoc/Hospital_detection',
+        documentation: 'https://github.com/geospoc/Hospital_detection',
```
Comment (vinayakkulkarni) [inline content/en/1.index.md:19]:
```suggestion
        documentation: 'https://github.com/geospoc/gqi-mapper-hospital-detection#geospatial-hospital-detection',
```

File: content/en/1.index.md [hunk 2]
```diff
@@ -11,14 +11,14 @@ bulletLinks:
...
-        crowdSourceValidationPlatform: 'https://unc-sch-dashboard-dev.geospoc.io/onboarding',
+        crowdSourceValidationPlatform: 'https://gqi-mapper-csp-prod.geospoc.com/',
       },
     'hospitals':
       {
-        codebase: '#',
-        documentation: '#',
-        portal: 'https://gqi-mapper-frontend-test.geospoc.io/dashboard',
-        crowdSourceValidationPlatform: 'https://gqi-mapper-csp-test.geospoc.io/',
+        codebase: 'https://github.com/geospoc/Hospital_detection',
```
Comment (vinayakkulkarni) [inline content/en/1.index.md:18]:
```suggestion
        codebase: 'https://github.com/geospoc/gqi-mapper-hospital-detection',
```

**Review by vinayakkulkarni — CHANGES_REQUESTED** [review]

File: content/en/1.index.md (hunk 1 above)
Comment (tusharkgeospoc) [reply content/en/1.index.md]: Updated under commit - https://github.com/geospoc/gqi-mapper-documentation/pull/115/commits/bb5f6e7334dca9c6f340254370825f43bfe02969

File: content/en/1.index.md (hunk 2 above)
Comment (tusharkgeospoc) [reply content/en/1.index.md]: Updated under commit - https://github.com/geospoc/gqi-mapper-documentation/pull/115/commits/bb5f6e7334dca9c6f340254370825f43bfe02969

**Review by vinayakkulkarni — APPROVED** [review]

**Pull request merged**